esmvaltool/recipes/ref/recipe_ref_sea_ice_area_basic.yml

```diff
@@ -0,0 +1,168 @@
+      end_longitude: 360
+      start_latitude: -90
+      end_latitude: -30
+    climate_statistics:
+      period: month
+    area_statistics:
+      operator: sum
+    convert_units:
+      units: 1e6 km2
+
+  pp_sh_feb:
+    extract_region:
+      start_longitude: 0
+      end_longitude: 360
+      start_latitude: -90
+      end_latitude: -30
+    extract_month:
+      month: 2
+    area_statistics:
+      operator: sum
+    convert_units:
+      units: 1e6 km2
+
+
+diagnostics:
+
+  siarea_seas:
+    description: Create seasonal cycle plot of sea ice area.
+    variables:
+      sea_ice_area_nh: &var_basic
+        short_name: siconc
+        timerange: '1995/2014'
+        preprocessor: pp_nh
+        project: CMIP6
+        mip: SImon
+        exp: historical
+        ensemble: r1i1p1f1
+        grid: gr
+        reference_dataset: OSI-450-nh
+        additional_datasets:
+        - {dataset: OSI-450-nh, project: OBS, type: reanaly, version: v3, mip: OImon, tier: 2,
+           supplementary_variables: [{short_name: areacello, mip: fx}]}
+#        - {dataset: ESACCI-SEAICE, project: OBS6, tier: 2, type: sat, version: L4-SICONC-RE-SSMI-12.5kmEASE2-fv3.0-SH,
+#           supplementary_variables: [{short_name: areacello, mip: Ofx}]}
+      sea_ice_area_sh:
+        <<: *var_basic
+        preprocessor: pp_sh
+        reference_dataset: OSI-450-sh
+        additional_datasets:
+        - {dataset: OSI-450-sh, project: OBS, type: reanaly, version: v3, mip: OImon, tier: 2,
+           supplementary_variables: [{short_name: areacello, mip: fx}]}
+    scripts:
+      allplots:
+        script: monitor/multi_datasets.py
+        plot_folder: '{plot_dir}'
+        group_variables_by: variable_group
+        facet_used_for_labels: dataset
+        plots:
+          annual_cycle: &plot_basic
+            annual_mean_kwargs: False
+            pyplot_kwargs:
+              title: 'Sea ice area (million km2)'
+            plot_kwargs:
+              'MPI-ESM1-2-LR':
+                color: red
+                label: '{dataset}'
+                linestyle: '-'
+                linewidth: 2
+                zorder: 4
+              ESACCI-SEAICE:
+                color: black
+                label: '{dataset}'
+                linestyle: '-'
+                linewidth: 2
+                zorder: 3
+              OSI-450-nh:
+                color: blue
+                label: 'OSISAF/CCI'
+                linestyle: '-'
+                linewidth: 2
+                zorder: 3
+              OSI-450-sh:
+                color: blue
+                label: 'OSISAF/CCI'
+                linestyle: '-'
+                linewidth: 2
+                zorder: 3
+              default:
+                color: lightgray
+                label: null
+                linestyle: '-'
+                linewidth: 1
+                zorder: 1
+
+  siarea_min:
+    description: Create time series of September/March sea ice area.
+    variables:
```

Comment:
I think this should be "Create time series of September/February sea ice area"

Reply:
Thanks for spotting this. Changed here: https://github.com/ESMValGroup/ESMValTool/pull/3891/commits/87b2ad56a5c9812c2550abae22bbb3972e310d09